app/views/lists/show.html.erb

```diff
@@ -11,10 +11,13 @@
 </p>
 
 <ul>
-  <% @list.papers.each do |paper| %>
-    <li>
-      <%=paper.title%>
-    </li>
+  <% @list.references.each do |reference| %>
+    <%if reference.paper.present?%>
+      <li>
+      <%=reference.paper.title%> 
+      <%= link_to 'Remove', reference_path(reference) %> 
```

Comment:
Does this work? And if so, why does it work? I'd expect you to need to send a `method: :delete` somewhere in there, depending on how the routes are wired.